core/build.sbt

```diff
@@ -431,9 +430,11 @@ lazy val operator =
         val targetDir    = "/app"
 
         new Dockerfile {
-          from("adoptopenjdk/openjdk8:latest")
+          from("adoptopenjdk/openjdk8:alpine")
```

Comment:
Is there a version of Alpine we can specify here?

Reply:
AFAIK that adoptopenjdk/openjdk8:alpine points to a 3.12

https://github.com/AdoptOpenJDK/openjdk-docker/blob/master/8/jdk/alpine/Dockerfile.hotspot.releases.full

Not sure we can add anything else @yuchaoran2011

Reply:
You verified that Alpine 3.9 didn't have critical vulnerabilities. Is that the case for 3.12 too?

Reply:
Yep, I run the scan against this very image: adoptopenjdk/openjdk8:alpine